src/stateFromElement.js

```diff
@@ -260,6 +276,9 @@ class BlockGenerator {
     if (element.childNodes != null) {
       Array.from(element.childNodes).forEach(this.processNode, this);
     }
+    if (SELF_CLOSING_ELEMENTS.hasOwnProperty(tagName)) {
+      this.processText('~');
```

Comment:
If you can't get to it, no big deal, I can come up with something when the need arises.